design/common.blocks/popup/_theme/popup_theme_islands.styl

```diff
@@ -8,9 +8,9 @@
     animation-duration: 0.1s;
     animation-timing-function: ease-out;
 
+    border: 1px solid rgba(0, 0, 0, 0.1);
     background: #fff;
-    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1),
-                0 10px 20px -5px rgba(0, 0, 0, 0.4);
+    box-shadow: 0 10px 20px -5px rgba(0, 0, 0, 0.4);
```

Comment:
@mishanga Do you remember why we use `box-shadow` instead of `border`? It feels that it was necessary for some reasons, wasn't it?